### PR TITLE
Title

### PR DESCRIPTION
**Pull request by gabriel-v**
core

### CODE REVIEW

File: liquidcore/home/templates/base.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!doctype html>
 <meta charset="utf-8">
-<title>Liquid Investigations</title>
+<title>{{ liquid_title }} - Liquid Investigations</title>
```
Comment (mgax) [inline liquidcore/home/templates/base.html:3]: Maybe keep just `{{ liquid_title }}`?

File: liquidcore/home/templates/homepage.html [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {% extends "base.html" %}
 
 {% block content %}
-<h1>Liquid Investigations</h1>
+<h1 class="shady">Liquid Investigations</h1>
+<h1>{{ liquid_title }}</h1>
```
Comment (mgax) [inline liquidcore/home/templates/homepage.html:5]: Yeah that's super impolite. Use a single `<h1>` element within the page. The vertical one can be a `<p>`, or better yet, [`<aside>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside).

File: liquidcore/home/templates/homepage.html [hunk 3]
```diff
@@ -46,3 +47,13 @@ <h1>Liquid Investigations</h1>
 {% endif %}
 
 {% endblock %}
+{% block headstyles %}
+<style>
+.shady {
+  transform: rotate(90deg);
```
Comment (mgax) [inline liquidcore/home/templates/homepage.html:53]: Sure.

**Review by mgax — APPROVED** [review]

File: liquidcore/home/templates/homepage.html (hunk 2 above)
Comment (gabriel-v) [reply liquidcore/home/templates/homepage.html:5]: this will get replaced with a decent theme anyway

**Pull request merged**